ee/clickhouse/views/experiments.py

```diff
@@ -232,6 +233,40 @@ def validate_saved_metrics_ids(self, value):
 
         return value
 
+    def validate_metrics(self, value):
+        # TODO: This isn't correct most probably, we wouldn't have experiment_id inside ExperimentTrendsQuery
```

Comment:
@jurajmajerik do you know this? What shape does the frontend send `metrics` in?

Reply:
not implemented yet, but it will be:

```
{
	type: "primary" | "secondary",
	query: ExperimentTrendsQuery | ExperimentFunnelsQuery
}[]
```

Reply:
ok if I leave this as is for now and we update when metrics do come in and we can write tests? Shape should be the same, only change would be what we validate i guess